tensorflow/compiler/xla/service/gpu/target_util.cc

```diff
@@ -222,12 +230,12 @@ llvm::Value* EmitCallToTargetFunction(
           llvm_ir::PrimitiveTypeToIrType(to_type, module));
     }
     llvm::FunctionType* callee_type = llvm::FunctionType::get(
-        llvm_ir::PrimitiveTypeToIrType(output_type, module),  // Return type.
+        llvm_ir::PrimitiveTypeToIrType(gpu_info->target_function_info.result_type, module),  // Return type.
```

Comment:
@sumesh13 you might want to run a `git clang-format`?

Reply:
I did do the format. Let me redo it. I have one more set of changes that cleans up some dependency thats not needed anymore. Will do the clang-format along with that